src/align.rs

```diff
@@ -148,7 +150,7 @@ impl<T: Copy, C: AlignedByteChunk> AlignedVec<T, C> {
         }
     }
 
-    /// Returns the number of elements in the vector.
+    /// Return the number of elements in the vector.
```

Comment:
Nit: Convention for doc comments is to use present tense, so this should be "Returns" instead of "Return" (the way it was before). See for example [the `slice::len` method in the std docs](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.len). Not really an important thing to clean up, but we should stick with the more conventional wording in our comments.

Reply:
Oh, oops, I thought it was the other way around.